quickstarts/blazor-web-assembly/config.yml

```diff
@@ -0,0 +1,55 @@
+
+
+slug: blazor-web-assembly
+description: |
+  ## Comprehensive monitoring quickstart for Blazor Web Assembly
+
+  Microsoft developed Blazor Web Assembly to build web UIs with C#, HTML, and CSS. As a feature of ASP.NET, our Blazor Web Assembly integration with the Browser Monitoring agent lets you monitor your apps with our quickstart dashboards right out of the box.
+
+
+  ## Why monitor Blazor Web Assembly?
+
+  Blazor Web Assembly offers a lot of useful features in addition to allowing C# developers to utilize their preferred language for web development. Blazor Web Assembly allows you to run your code through WebAssembly on the server or the client, and you can decide which users or devices experience each method.
+  
+  There are a few compelling arguments that support the employment of Blazor Web Assembly:
+  - All of the major browsers support WebAssembly.
+  - Using C# for interactive web apps.
+  - Utilize current C# libraries.
+  - Near-native performance.
+  - Debugging and tooling.
+
+  ## What’s included in this quickstart?
+
+  Get these Blazor Web Assembly monitoring features out of the box with our quickstart:
+  - Understand the health of your system with dashboards that show throughput and error rate data, logs, web transaction time, page views, page load time, and more. 
+  - Alerts ()
+  - Monitor scripts and functions.
+  - Monitor web transactions.
+
+summary: | 
+  Instrument and monitor your Blazor Web Assembly application with our Browser Monitoring agent. Analyze your performance data right out of the box.
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: Blazor Web Assembly
+documentation:
+  - name: Blazor Web Assembly installation docs
+    description: |
+      Our Browser Monitoring agent can be used to instrument and monitor your Blazor Web Assembly application. Perform an immediate analysis of your performance statistics. 
+    url: >- 
+      https://docs.newrelic.com/
+keywords:  
+  - apm
+  - dotnet
+  - .net
+  - blazor
+  - c-sharp
+  - c#
```

Comment:
can remove

Reply:
Hi @pkosanam, removed. Kindly review it.